dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/group/ComponentGroupApiImpl.kt

```diff
@@ -52,15 +66,20 @@ class ComponentGroupApiImpl(var parent: FieldNodeParent? = null) : ComponentGrou
         return null
     }
 
-    override fun <T : ComponentBase> create(identifier: String, clazz: KClass<T>, init: (T.() -> Unit)?): T {
+    override fun <T : ComponentBase> create(
+        identifier: String,
+        insertBeforeIdentifier: String?,
```

Comment:
Same question as in the other file; do you have an example for this insertBeforeIdentifier?

Reply:
Answered in other comment.

Reply:
ok